pkg/services/ldap/ldap.go

```diff
@@ -46,6 +46,9 @@ type Server struct {
 // Bind authenticates the connection with the LDAP server
 // - with the username and password setup in the config
 // - or, anonymously
+//
+// Dial() sets the connection with the server for this Struct. Therefore, we require a
```

Comment:
I would specify documentation only once, either in Dial or in package desc. I would expect our LDAP module to resemble logic of any db connector. 

So for me, that’s not really surprising, so I am bias here